sample-configs/run_config_softmax_test.json

```diff
@@ -1,30 +1,12 @@
 {
```

Comment:
No input shape for either ort web or tfjs?

Reply:
Same as the above.